.github/workflows/dapr.yml

```diff
@@ -173,4 +173,9 @@ jobs:
             --body "${RELEASE_BODY}" \
             --prerelease true \
             ${RELEASE_ARTIFACT[*]}
-
+      - name: Upload Helm charts to Helm Registry
+        if: startswith(github.ref, 'refs/tags/v')
+        run: |
+          export HELM_EXPERIMENTAL_OCI=1
```

Comment:
We do not need this because makefile target will set this env var.

Reply:
makefile target is invoked after the login. So need it here as well since the login command needs this to be set